web/viewer.css

```diff
@@ -920,6 +920,24 @@ html[dir="rtl"] #viewOutline.toolbarButton::before {
   content: url(images/toolbarButton-search.png);
 }
 
+.toolbarButton.pdfSidebarNotification::after {
+  position: absolute;
+  display: inline-block;
+  top: 0px;
+  /* Create a filled circle, with a diameter of 10 pixels, using only CSS: */
+  content: '';
+  background-color: #8BE651;
+  height: 10px;
```

Comment:
I've updated the size to `9px` and changed the colour slightly (such that it's now using a value found in http://firefoxux.github.io/StyleGuide/#/colours).

Also, after going back-and-forth a bit on the tooltip, I figured that it probably can't hurt to add one.
My line of reasoning here is that while we know the context here (i.e. from this PR), a user that sees the notification may find the tooltip helpful to explain what the notification is about.
However, it's easy to change or even remove the tooltip, if we decide to do so later on.
@timvandermeij Does the implementation seem reasonable here, in your opinion?